python_modules/dagit/dagit/cli.py

```diff
@@ -86,6 +87,13 @@ def create_dagit_cli():
     type=click.INT,
     show_default=True,
 )
+@click.option(
+    "--db-pool-recycle",
+    help="The maximum age of a connection to use from the sqlalchemy pool without connection recycling. Set to -1 to disable. Not respected in all configurations.",
```

Comment:
> Set to -1 to disable

is this how it works at the sqlalchemy level?

Reply:
Yup.

> this setting causes the pool to recycle connections after the given number of seconds has passed. It defaults to -1, or no timeout.